.travis.yml

```diff
@@ -22,23 +27,23 @@ branches:
     - master
 
 before_script:
+  - export JULIA_PROJECT=${HOME}/.julia/environments/v1.0/
```

Comment:
Hm, couldn't you simply `unset JULIA_PROJECT`, to disable the use of a custom Julia project completely?

Reply:
Yep, would also be possible.